docs/README.md

```diff
@@ -1,20 +1,121 @@
-# User Guide
+# Duke User Guide
+
+## Introduction
+Duke is a personal task manager that helps you track your todos, events, and
+deadlines. It is in the form of a chatbot and can be used on the command line
+as well as from a GUI.
+
+## Quick Start
+1. Ensure you have Java 11 or above installed in your Computer.
+2. Download the latest `duke.jar` [here](https://github.com/gary-lgy/duke/releases).
+4. Double-click the file to start the app. The GUI should appear in a few seconds.
+5. Type the command in the command box and press Enter to execute it.  
+   e.g. typing `help` and pressing Enter will open the help window.
+6. Some example commands you can try:
+    - `list` : lists all tasks
+    - `todo learn to use Duke` : adds a todo
+    - `done 1` : mark the first task as done
+    - `bye` : exits the app
+
+Details of the commands are explained below.
 
 ## Features 
 
-### Feature 1 
-Description of feature.
+### Available in both CLI and GUI
+GUI:
+
+![GUI version](./gui_demo.gif)
+
+CLI:
+
+![CLI version](./cli_demo.gif)
+
+While the GUI may be more intuitive to use, the CLI version provides a simpler
```

Comment:
a clear readme overall! :) having both GUI and CLI accessible from the same JAR file is quite a good idea!